.vscodeignore

```diff
@@ -7,8 +7,7 @@
 .yarn/**
 docs/**
 emoji/**
-images/docs/sponsors/**
-images/docs/gitlens-preview.gif
+images/docs
```

Comment:
We can't exclude all the docs images, because any images referenced in the README (which is the extension manifest page), walkthroughs, etc, need to be available locally.

Reply:
>  any images referenced in the README

They are referenced to raw.githubusercontent.com, for example: https://github.com/gitkraken/vscode-gitlens/blob/0f6f6ac6fa05b6d6be1d225d6cc973dcfcd2a60e/README.md?plain=1#L14

If I open readme, I see that it's referenced to cloud storage, not to locally bundled images

![Screenshot 2024-12-18 at 10 27 12](https://github.com/user-attachments/assets/c9f3f7db-41c7-4704-a1d1-21685fde0710)

There is no place where the image from this dir is used locally

> walkthroughs, etc

walkthroughs assets are placed in another directory, I didn't touch it. But it can also be moved to cloud storage and removed from the bundle

Reply:
Ah -- I forgot that the VS Code marketplace changed to require all README links to be `https`. So it looks like you are correct those images don't need to be bundled.

As for the walkthroughs, if we use `https` links then the walkthroughs won't work offline, which is not ideal. So we should leave them alone.